models/public/yolo-v4-tf/accuracy-check.yml

```
@@ -29,20 +30,26 @@ models:
           - conv2d_109/BiasAdd
     datasets:
       - name: ms_coco_detection_80_class_without_background
+        im_reader: pillow
```

there is not im_reader field in AC
```suggestion
        reader: pillow
```